### DISCUSSION TITLE
How to enable s3 support instead of minio for milvus v2.0.0-rc2?

### DISCUSSION THREAD

**Discussion by sounak07** [milvus-io/milvus#6980] [Q&A and General discussion]
I'm trying to use s3 as object storage instead of minio but I'm unable to find the proper flags in docker-compose or configs to enable that. If anyone can point me where I should look for the flag or what are the right flags for this that would be great. I have gone the code base but have not been able to come up with useful information regarding this.
Thank you

**Comment by xiaofan-luan** [comment]
To start Milvus Cluster using S3 instead of minio as storage, you need to add some parameters at startup time. Milvus helm configuration parameters can be found in the documentation: https://artifacthub.io/packages/helm/milvus/milvus

**Comment by xiaofan-luan** [comment]
![image](https://user-images.githubusercontent.com/83447078/128824326-793a32a8-2ec2-4661-91ff-e20ab4c672f7.png)

for example

helm install lym-release milvus/milvus --set cluster.enabled=true --set externalS3.enabled=true --set externalS3.host='s3.ap-southeast-1.amazonaws.com' --set externalS3.port=80 --set externalS3.accessKey=accesskey --set externalS3.secretKey=secret --set externalS3.bucketName=uso --set minio.enabled=False --set service.type=LoadBalancer
